ui/enhancement_shaman/presets.ts

```diff
@@ -46,6 +50,11 @@ export const DefaultRotation = EnhancementShamanRotation.create({
 		fire: FireTotem.MagmaTotem,
 		water: WaterTotem.ManaSpringTotem,
 	}),
+  maelstromweaponMinStack: 3,
+  lightningboltWeave: true,
+  lavaburstWeave: false,
```

Comment:
this might not be needed, by default it's false if left out.

Reply:
imo it doesnt hurt to have it, just for clarity reasons